en/_posts/2016-04-20-quick-start-with-marathon.md

```diff
@@ -6,16 +6,16 @@ author: devatoria
 lang: en
 ---
 
-Welcome to the quick start tutorial of the PaaS Docker lab. This document will guide you to launch your first application and play with it.
+Welcome to the quick start tutorial of the PaaS Docker lab. This document will help you launch your first application using the Marathon Web UI.
```

Comment:
Unless "PaaS Docker" is an official name, I'd recommend the "Docker PaaS" formulation to sound less frenchy ;)